### PR TITLE
Replace Assessment screen with ThankYou screen

### LINKED ISSUE #145: [Symptom Report] Replace Assessment screen with Thank You screen
Requirements:
After completing a Symptom Report, the user should be directed to a Thank You screen
- “Thank you for submitting your data and supporting the COVID-19 response effort”
- Call to action to return to the app on the following day
- Call to action to share the app
- Button to return to the Home Screen

Acceptance criteria:
- [x] The `Assessment` model has been deleted and all references to it have been removed.
- [x] The positive and negative assessment widgets have been removed.
- [x] The Assessment screen/route has been deleted. 
- [x] A new Thank You screen has been added to the end of the Symptom Report flow (was checkup) and matches the requirements expressed above.

Wireframe:
Record Your Symptoms

### PR DESCRIPTION
**Pull request by goderbauer**
Fixes https://github.com/coronavirus-diary/coronavirus-diary/issues/145.

screencap

This PR is also exposing a flaw in the current bloc model: https://github.com/coronavirus-diary/coronavirus-diary/issues/146#issuecomment-606285578

### CODE REVIEW

File: lib/src/ui/screens/checkup/checkup.dart [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import 'package:covidnearme/src/blocs/questions/questions.dart';
 import 'package:covidnearme/src/data/repositories/questions.dart';
+import 'package:covidnearme/src/ui/screens/thankyou/thankyou.dart';
```
Comment (ferndot) [inline lib/src/ui/screens/checkup/checkup.dart]: Non-blocking: you could instead import the router here.
```suggestion
import 'package:covidnearme/src/ui/router.dart';
```

File: lib/src/ui/screens/checkup/checkup.dart [hunk 2]
```diff
@@ -94,25 +94,10 @@ class _CheckupScreenBodyState extends State<CheckupScreenBody> {
     PreferencesState preferencesState,
```
Comment (ferndot) [inline lib/src/ui/screens/checkup/checkup.dart]: It looks like the preferences bloc is not needed anymore in this screen. Could you remove it please?

File: lib/src/ui/screens/thankyou/thankyou.dart [hunk 3]
```diff
@@ -0,0 +1,91 @@
+import 'package:covidnearme/src/ui/widgets/network_unavailable_banner.dart';
```
Comment (ferndot) [inline lib/src/ui/screens/thankyou/thankyou.dart]: Could you move this file into `screens/checkup`. It is a part of the Symptom Report (checkup) flow now.

File: lib/src/ui/screens/thankyou/thankyou.dart [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+import 'package:covidnearme/src/ui/widgets/share.dart';
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+
+import 'package:covidnearme/src/l10n/app_localizations.dart';
+import 'package:covidnearme/src/ui/widgets/scrollable_body.dart';
+import 'package:covidnearme/src/ui/widgets/stay_safe.dart';
+
+class ThankYouScreen extends StatefulWidget {
+  static const routeName = '/thankyou';
```
Comment (ferndot) [inline lib/src/ui/screens/thankyou/thankyou.dart]: Could this be nested under the Symptom Report route? I'm not sure whether snake or kebab case is more appropriate here.
```suggestion
  static const routeName = 'symptom-report/thankyou';
```

**Review by ferndot — CHANGES_REQUESTED** [review]

**Comment by ferndot** [comment]
#155 just landed and may require a few changes to this.

Lots of moving pieces all at once :D

File: lib/src/l10n/app_en.arb [hunk 5]
```diff
@@ -16,49 +16,34 @@
...
+    "description": "Asking the user to report their symptoms again tomorrow"
   },
 
   "checkupScreenErrorRetrievingExperience": "There was an error retrieving the checkup experience. Please try again later.",
   "@checkupScreenErrorRetrievingExperience": {
     "description": "An error occurred while trying to fetch data"
   },
 
-  "checkupScreenHUDLabel": "Loading your assessment",
+  "checkupScreenHUDLabel": "Submitting your report",
```
Comment (ferndot) [inline lib/src/l10n/app_en.arb]: We should replace `checkup`->`symptomReport` in a follow-up PR.

File: lib/src/ui/screens/symptom_report/symptom_report.dart [hunk 6]
```diff
@@ -92,33 +91,18 @@ class _SymptomReportScreenBodyState extends State<SymptomReportScreenBody> {
...
-        symptomReportState.assessment) {
-      Preferences newPreferences = preferencesState.preferences.cloneWith(
-        lastAssessment: symptomReportState.assessment,
-        // If they've completed an assessment, then don't show them the welcome
-        // screen again.
-        completedTutorial: true,
-      );
-      context.bloc<PreferencesBloc>().add(UpdatePreferences(newPreferences));
-    }
+    // TODO(goderbauer): Reset the SymptomsReport bloc.
```
Comment (ferndot) [inline lib/src/ui/screens/symptom_report/symptom_report.dart]: https://github.com/coronavirus-diary/coronavirus-diary/issues/171

Comment (goderbauer) [reply lib/src/ui/screens/symptom_report/symptom_report.dart]: Added a link to that bug.

**Review by ferndot — DISMISSED** [review]
LGTM!

File: lib/src/l10n/app_en.arb (hunk 5 above)
Comment (goderbauer) [reply lib/src/l10n/app_en.arb]: Yeah, there are many other strings that still have the name checkup in them. Let's do that in a separate PR.

File: lib/src/ui/screens/symptom_report/symptom_report.dart (hunk 6 above)
Comment (ferndot) [reply lib/src/ui/screens/symptom_report/symptom_report.dart]: 👍

File: lib/src/l10n/app_en.arb (hunk 5 above)
Comment (ferndot) [reply lib/src/l10n/app_en.arb]: Sounds good!

**Comment by goderbauer** [comment]
@joshua-s I had to resolve another merge conflict, which dismissed your review. Can you approve again and possibly also submit to lower the chances of yet another merge conflict? Thanks!

**Review by dnfield — APPROVED** [review]

**Pull request merged**